pySDC/projects/AllenCahn_Bayreuth/run_simple_forcing_benchmark.py

```diff
@@ -1,7 +1,9 @@
 from argparse import ArgumentParser
+import warnings
 import numpy as np
 from mpi4py import MPI
 
+warnings.filterwarnings("ignore")
```

Comment:
What's that about?

Reply:
If not, the warning message of deprecation for the "old" classes is printed in the error of the mpi process, which fails the test